styles/globals.css

```diff
@@ -6,9 +6,9 @@ body {
   font-size: 16px;
   font-weight: 400;
   line-height: 20px;
-
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
```

Comment:
❓ Hey @nachoiacovino! Would you mind explaining the reasoning behind this change? For this simple starter app, it mainly affects the distances between the card containers but if that's what you wanted to achieve I'd prefer keeping the global box-sizing rule and fixing the components locally

Reply:
It was breaking the html tag to have it global, if you check the "before" mobile screenshot, everything was pushed a little bit to the left, like having an invisible margin-right. This fixed it.

If you find another way of fixing it I would love to hear it 😁